src/python/library/tritonclient/http/__init__.py

```diff
@@ -1757,7 +1757,7 @@ def set_shared_memory(self, region_name, byte_size, offset=0):
         self._parameters['shared_memory_region'] = region_name
         self._parameters['shared_memory_byte_size'] = byte_size
         if offset != 0:
-            self._parameters['shared_memory_offset'].int64_param = offset
```

Comment:
Why are you removing int64_param?

Reply:
`int64_param` is the specification on GRPC side, HTTP doesn't use that and specifying `int64_param` is likely to result in "malformed" JSON body